tests/e2e/flowtest/test_cephfilesystem_creation.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.helpers.helpers import create_resource, validate_cephfilesystem
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.utility import templating
+from ocs_ci.ocs.ocp import OCP
+
+
+logger = logging.getLogger(__name__)
+cephfs_data = templating.load_yaml(constants.CEPHFILESYSTEM_YAML)
+cephfs_data["metadata"]["name"] = "test-ceph-fs"
+cephfs_data["metadata"]["namespace"] = defaults.ROOK_CLUSTER_NAMESPACE
+cephfs_data["metadata"]["labels"] = {"use": "test"}
+
+
+class TestCephFileSystemCreation:
+    """
+    Testing Creation of a filesystem and checking its existence.
+    Also checking if the same filesystem can be created more than once or not.
```

Comment:
Also checking if the same filesystem can't  be created twice.

Reply:
@shyRozen I have made the changes, could you please review it.